src/openforms/forms/models/form_variable.py

```diff
@@ -205,19 +205,15 @@ def __str__(self):
         return _("Form variable '{key}'").format(key=self.key)
 
     def get_initial_value(self):
-        if self.source == FormVariableSources.static:
-            return STATIC_INITIAL_VALUES[self.initial_value]()
-        else:
-            return self.initial_value or None
+        return self.initial_value or None
 
     @staticmethod
-    def get_default_static_variables() -> List["FormVariable"]:
-        now = FormVariable(
-            name="Now",
-            key="now",
-            source=FormVariableSources.static,
-            data_type=FormVariableDataTypes.datetime,
-            initial_value="now",
-        )
+    def get_static_data() -> List[dict]:
+        now = {
+            "name": "Now",
+            "key": "now",
+            "data_type": FormVariableDataTypes.datetime,
+            "initial_value": STATIC_INITIAL_VALUES["now"](),
+        }
```

Comment:
just a comment - but I think I'd still wrap this as `FormVariable` instances with all the convenience methods, especially in light of #1708 and #1711 where we probably just want to define a python list of `FormVariable` instances.